packages/header/src/Header.tsx

```diff
@@ -0,0 +1,71 @@
+import { CoreProvider } from '@ag.ds-next/core';
+import { Header as AgDsHeader } from '@ag.ds-next/header';
+import { AvatarIcon } from '@ag.ds-next/icon';
+import { MainNav, MainNavButton } from '@ag.ds-next/main-nav';
+import { Logo } from '@ag.ds-next/ag-branding';
+
+export type HeaderProps = {
+	authenticated: boolean;
+	activePath: string;
+	handleSignIn?: React.MouseEventHandler<HTMLButtonElement>;
```

Comment:
what do you think about this -- Is the consumer of this component concerned with the triggering element?

the original promise<void> signaled to the user the intent of how the property should be used. 

interested to hear your thoughts